### PR TITLE
MM-35932: changes strings due to re-packaging

### PR DESCRIPTION
**Pull request by koox00**
#### Summary

Updates in-product strings referencing E10 & E20 to new packaging.

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-35932

#### Related Pull Requests

- Has server changes (please link here)
- Has redux changes (please link here)
- Has mobile changes (please link here)

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: components/admin_console/feature_discovery/features/permissions.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ const PermissionsFeatureDiscovery: React.FC = () => {
         <FeatureDiscovery
             featureName='permissions'
             titleID='admin.permissions_feature_discovery.title'
-            titleDefault='Set role-based permissions with Mattermost Enterprise E10'
+            titleDefault='Set team-based permissions with Mattermost Enterprise'
```
Comment (wiersgallak) [inline components/admin_console/feature_discovery/features/permissions.tsx:17]: This should be 'Set team-based permissions with Mattermost Professional`

File: components/admin_console/feature_discovery/features/announcement_banner.tsx [hunk 2]
```diff
@@ -14,7 +14,7 @@ const AnnouncementBannerFeatureDiscovery: React.FC = () => {
         <FeatureDiscovery
             featureName='announcement_banner'
             titleID='admin.announcement_banner_feature_discovery.title'
-            titleDefault='Create custom announcement banners with Mattermost Enterprise E10'
+            titleDefault='Create custom announcement banners with Mattermost Enterprise'
```
Comment (wiersgallak) [inline components/admin_console/feature_discovery/features/announcement_banner.tsx:17]:
```suggestion
            titleDefault='Create custom announcement banners with Mattermost Professional'
```

File: components/admin_console/feature_discovery/features/permissions.tsx (hunk 1 above)
Comment (wiersgallak) [inline components/admin_console/feature_discovery/features/permissions.tsx:17]:
```suggestion
            titleDefault='Set team-based permissions with Mattermost Professional'
```

File: i18n/en.json [hunk 3]
```diff
@@ -198,7 +198,7 @@
   "admin.advance.cluster": "High Availability",
   "admin.advance.metrics": "Performance Monitoring",
   "admin.announcement_banner_feature_discovery.copy": "Create announcement banners to notify all members of important information.",
-  "admin.announcement_banner_feature_discovery.title": "Create custom announcement banners with Mattermost Enterprise E10",
+  "admin.announcement_banner_feature_discovery.title": "Create custom announcement banners with Mattermost Enterprise",
```
Comment (wiersgallak) [inline i18n/en.json:201]:
```suggestion
  "admin.announcement_banner_feature_discovery.title": "Create custom announcement banners with Mattermost Professional",
```

File: i18n/en.json [hunk 4]
```diff
@@ -1380,7 +1380,7 @@
   "admin.password.symbol": "At least one symbol (e.g. \"~!@#$%^&*()\")",
   "admin.password.uppercase": "At least one uppercase letter",
   "admin.permissions_feature_discovery.copy": "Decide who can perform an array of actions such as creating channels, inviting people, managing and archiving channels, managing webhooks, and more in Permission Schemes.",
-  "admin.permissions_feature_discovery.title": "Set role-based permissions with Mattermost Enterprise E10",
+  "admin.permissions_feature_discovery.title": "Set team-based permissions with Mattermost Enterprise",
```
Comment (wiersgallak) [inline i18n/en.json:1383]:
```suggestion
  "admin.permissions_feature_discovery.title": "Set team-based permissions with Mattermost Professional",
```

**Review by jwilander — APPROVED** [review]
LGTM

**Review by wiersgallak — APPROVED** [review]
LGTM

**Review by hahmadia — APPROVED** [review]

**Review by michaelgamble — CHANGES_REQUESTED** [review]
Only one change that I see.. the font for headings aren't matching our updated design system / whats in the desing.

Currently the heading font is showing "Open Sans" - open sans is now only for body content.

The heading font should be "Metropolis"
image

**Review by michaelgamble — APPROVED** [review]
Looks good to me :)

**Pull request merged**